integration_tests/minimum_fee_test.go

```diff
@@ -23,9 +24,11 @@ func TestXionMinimumFeeDefault(t *testing.T) {
 	}
 
 	t.Parallel()
-	td := BuildXionChain(t, "0.0uxion", ModifyInterChainGenesis(ModifyInterChainGenesisFn{ModifyGenesisShortProposals}, [][]string{{votingPeriod, maxDepositPeriod}, {defaultMinGasPrices.String()}}))
+	td := BuildXionChain(t, "0.1uxion", ModifyInterChainGenesis(ModifyInterChainGenesisFn{ModifyGenesisShortProposals}, [][]string{{votingPeriod, maxDepositPeriod}, {defaultMinGasPrices.String()}}))
 
 	assertion := func(t *testing.T, ctx context.Context, xion *cosmos.CosmosChain, xionUser ibc.Wallet, recipientAddress string, fundAmount int64) {
+		fmt.Println("waiting check logs")
+		time.Sleep(30 * time.Second)
```

Comment:
Do we want this wait all the time?

Reply:
No, this was leftover from debugging. removed now.